package.json

```diff
@@ -15,7 +15,8 @@
     "test:watch": "node --test --watch",
     "prepare": "husky",
     "run": "node bin/cli.mjs",
-    "watch": "node --watch bin/cli.mjs"
+    "watch": "node --watch bin/cli.mjs",
+    "postinstall": "git-deps install"
```

Comment:
If this is the intended behavior and the solution, we should move this dependency to node.js organization, before merging this pull-request. It will run on every postinstall, and it's huge security concern.

Reply:
We are still looking for solutions, see #236

Reply:
Agreed with @anonrig although I'd say this is a draft, but we def... need to find a perma solution.

Reply:
We should at the very least move this dep to @pkgjs. @avivkeller feel free to open an issue on /admin to transfer this + ownership.

Reply:
Shouldn't we first decide whether this is a perma solution? I'd argue that we shouldn't maintain a package that we aren't going to use-but I'm happy to transfer it to the org regardless

Reply:
Sure, but this PR cannot land till we decide that, or at least cannot land whilst git-deps is a dependency here :sweat_smile:

Reply:
> Sure, but this PR cannot land till we decide that

Per the vote in https://github.com/nodejs/api-docs-tooling/issues/236, it looks like https://github.com/nodejs/nodejs.org/pull/7776 will resolve that, and once they are published, I'll switch it. `git-deps` allows me to test this PR until then.

Reply:
I'm resolving this disucssion, and adding the `blocked` label until the upstream PR lands.